src/static/json/resultList.json

```diff
@@ -2,7 +2,7 @@
     [{
```

Comment:
Can you find "" inside of Description and remove it?
It causes errors when html rendering

Reply:
Done! Replace " to &quot;